### DISCUSSION TITLE
profile picture is blur

### DISCUSSION THREAD

**Discussion by rksc143** [community/community#65722] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

why my profile picture is blur any idea 
![Screenshot 2023-09-02 073542](https://github.com/community/community/assets/139877906/bbfe34df-0257-4727-ae2c-00f3338d70cd)

**Comment by GhostOf0days** [comment, marked as answer]
Try reuploading the original or [upscale](https://imgupscaler.com/) it.

**Comment by rksc143** [comment]
Txx
